app/views/cdo/manage/live.njk

```diff
@@ -0,0 +1,80 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends '_layout.njk' %}
+
+{% block pageTitle %}{{ serviceName }}
+{% endblock %}
+
+{% block beforeContent %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      {{ govukBackLink({
+        text: "Back",
+        href: model.backLink,
+        attributes: {
+          id: "back"
+        }
+        }) 
+      }}
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-one-half">
+      <h1 class="govuk-heading-l">
+        Manage CDOs
+      </h1>
+    </div>
+
+    <div class="govuk-grid-column-one-half">
+      <!--<div class="govuk-button-group defra-align-right-simple">
+        <a href="/cdo/manage/interim{{ model.srcHashParam }}" role="button" draggable="false" class="govuk-button govuk-button--secondary" data-module="govuk-button">
+          Interim exemptions
+        </a>
+      </div> -->
+    </div>
+
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <ul class="govuk-tabs__list govuk-!-margin-bottom-7">
+        {% for tab in model.tabs %}
+        {% if tab.visible %}
+        <li class="govuk-tabs__list-item {% if tab.active %}govuk-tabs__list-item--selected{% endif %}">
+          <a class="govuk-tabs__tab" href="{{tab.href}}">
+            {{tab.label}}
+          </a>
+        </li>
+        {% endif %}
+        {% endfor %}
+      </ul>
+      <table class="govuk-table">
+        <caption class="govuk-table__caption govuk-visually-hidden">Active CDOs ordered by deadline</caption>
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header govuk-input--width-10" aria-sort="ascending">CDO expiry</th>
```

Comment:
I assume the aria-sort will need to be dynamic eventually, when full sorting added